apps/services/endorsements/api/src/app/modules/pagination/dto/pagination.dto.ts

```diff
@@ -0,0 +1,18 @@
+import { ApiProperty } from '@nestjs/swagger'
+import { IsNumberString, IsOptional, IsString } from 'class-validator'
+
+export class PaginationDto {
+  @ApiProperty({ required: false })
+  // @IsNumberString()
```

Comment:
Redundant comment? Use `IsNumber()` instead?

Reply:
fixed actually wanted the @IsNumberString() filter to address that